makefile

```diff
@@ -47,8 +47,6 @@ help:
 	@echo '      * rev -> prim'
 	@echo '      * fwd -> prim'
 	@echo '      * mix -> {rev, fwd, prim}'
-	@echo '      * within {prim, rev, fwd, mix}: mat -> arr -> scal'
-	@echo '      * only include {prim, rev, fwd, mix}/meta.hpp from the meta subfolders'
```

Comment:
This is unrelated, just removed it because its no longer true. We do not check for those subfolder as they no longer exist.

Reply:
I think the line 51 is still relevan (50 can be deleted)

Reply:
We no longer enforce that after the flatten was done. Its encouraged but not enforced.

Reply:
Ok, this is fine.